app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    vaccine_problems = 0
+    mask_problems = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError as ve:
+            print(ve)
+            vaccine_problems += 1
```

Comment:
![image](https://user-images.githubusercontent.com/58142395/216644218-79a537dd-464d-4d0b-b119-2408a40e941c.png)
So you can not use `vaccine_problems` and return result in this line